src/sync/pull-requests.test.ts

```diff
@@ -253,6 +255,13 @@ describe("sync/pull-request", () => {
 					.reply(200, pullRequest)
 					.get("/repos/integrations/test-repo-name/pulls/51/reviews")
 					.reply(200, reviewsPayload)
+					.get("/users/test-pull-request-reviewer-login")
+					.reply(200, {
+						login: "test-pull-request-reviewer-login",
+						avatar_url: "test-pull-request-reviewer-avatar",
+						html_url: "test-pull-request-reviewer-url",
+						email: "test-pull-request-reviewer-login@gmail.com"
```

Comment:
Please don't use `gmail.com` in any tests.  I've had issues in the past where test emails were sent to a real gmail address.  

Not that this would happen here since we're not sending email, but more of a piece of mind kind of thing.  please use something like `email.test` as `.test` is a reserved TLD that can't be used :)